rs/messaging/src/state_machine.rs

```diff
@@ -109,12 +109,13 @@ impl StateMachine for StateMachineImpl {
                 .observe_no_canister_allocation_range(&self.log, message);
         }
 
-        // Time out requests.
-        let timed_out_requests = state.time_out_requests();
+        // Time out expired messages.
+        // FIXME: We're now also timing out responses, update the relevant metrics.
```

Comment:
Based on the change, this `FIXME` seems to be a leftover.

Reply:
Also renamed the `timed_out_requests_total` metric to `timed_out_messages_total`. And dropped the FIXME,

Thanks for the catch.